ci/before_install.sh

```diff
@@ -5,6 +5,7 @@ set -ex
 
 macos_install() {
   [ "${CI-}" = true ] || brew bundle
+  brew install automake
```

Comment:
Instead of this just add it to the top-level Brewfile

Reply:
Wait I just realized it's already in there -_-. Must be something else going on here...

Reply:
I've been working on getting CI stable again with regards to the ruby-rnp failures, so I'll add this to the list. Hopefully I can get it finished today.

Reply:
Thanks, @dewyatt !

Reply:
Okay I see what's up here: https://github.blog/changelog/2020-04-15-github-actions-sets-the-ci-environment-variable-to-true/

Anyways, I'll get this stuff all fixed up.

Reply:
I'm not really sure why I wrote it that way originally anyways, it would only install the bundle if CI was not true (which worked because it used to be blank until yesterday). That'll teach me to comment my code I guess. I'll probably just replace that with `brew bundle`. Hmm.